Solutions/Corvus.Tenancy.Storage.Azure.Blob/Corvus/Tenancy/Storage/Azure/TenantProviderBlobStore.cs

```diff
@@ -166,14 +166,14 @@ public async Task<ITenant> CreateChildTenantAsync(string parentTenantId)
                 Tenant child = this.serviceProvider.GetRequiredService<Tenant>();
                 child.Id = parentTenantId.CreateChildId();
 
-                // At the least, we need to copy the default blob storage settings from its parent
+                // At the least, we need to copy the tenant blob storage settings from its parent
```

Comment:
I don't know if this change to the comments is quite sufficient.

I think I'd prefer it to state explicitly that we're only copying the tenant blob storage settings for the `corvustenancy` container (i.e., the one described by `ContainerDefinition`). (And this is a change, right. Before, this was actually copying non-container-specific default blob storage settings.)

Second, I'm not sure that it's really true that we "would expect this to be overridden by clients that wanted to establish their own settings". We have the ability for clients to be able to specify that they want to use a different storage account for all their child tenants, but I think most of the time we're going to expect clients just to use the same storage account as everyone else for their tenant settings. (I'm not even sure, given our newly defined approach to tenancy settings, whether we really even offer client-tenancy-settings isolation: we can store a client's child tenant settings in its own storage account, but all of the corresponding on-behalf-of tenants actually live under the relevant service tenants, which will not be using client-specific storage. So the reality is that even if a customer brings their own storage for tenancy, we'll still end up copying their settings into the common storage account any time we create an on-behalf-of tenant. So I think this idea of tenant-specific storage for tenancy settings may in effect be a fiction. This makes me wonder whether we should actually dramatically simplify this whole thing and say that the `TenantProviderBlobStore` just uses the storage account it's told to, and to remove all the code that walks through the tenant tree working out which store to use for any particular child tenant.)

Reply:
I've rejigged the comment. As discussed, the larger point will be dealt with via a separate discussion/issue.